modules/about-must-gather.adoc

```diff
@@ -27,8 +27,7 @@ For example:
 +
```

Comment:
This is outside of scope, but if possible, I would replace `For example:` with `.Example command` for the consistency. I would make that change in this whole section if possible. Thank you!

Reply:
@eromanova97 Thanks for the review comments. I agree.

@joaedwar Please get this addressed.